fastlane/Fastfile

```diff
@@ -23,13 +23,12 @@ private_lane :merged_prs_since_last_release do |options|
     "master"
   end
 
-  
-  # Find all PRs that have been merged to master since the last beta went out
+  # Find all PRs that have been merged since the last beta went out
   closed_prs = github_api(
     server_url: "https://api.github.com",
     api_token: github_token,
     http_method: "GET",
-    path: "/repos/guardian/#{repo}/pulls?state=closed&base=#{base_branch_name}&sort=created&direction=desc&per_page=100"
+    path: "/repos/guardian/#{repo}/pulls?state=closed&sort=updated&direction=desc&per_page=100"
```

Comment:
Is there a particular reason we're changing from `sort=created` to `sort=updated` here?

Reply:
In the past I have seen various batch jobs (e.g. mass deleting old branches) causing updates on very old PRs. So if possible, I would recommend leaving this as `sort=created`.

Reply:
I changed it because I assumed that would give us 'better' ordering within the PRs list (for some definition of 'better') but yes, you're right, I could absolutely see very old PRs being updated - maybe even this script/updating labels would do it? What we really need is a `sort=merged` but that doesn't seem to be an option. I'll swap it back.